google-cloud-spanner/src/test/java/com/google/cloud/spanner/GceTestEnvConfig.java

```diff
@@ -58,8 +58,10 @@ public class GceTestEnvConfig implements TestEnvConfig {
   private final SpannerOptions options;
 
   public GceTestEnvConfig() {
-    String projectId = System.getProperty(GCE_PROJECT_ID, "");
-    String serverUrl = System.getProperty(GCE_SERVER_URL, "");
+    // String projectId = System.getProperty(GCE_PROJECT_ID, "");
```

Comment:
nit: add a TODO here + a Do not merge label in the PR itself to make sure this is modified before the PR is merged.